ui/apps/ui/src/app/components/sort-by-functionality/sort-value-default.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_SORT = 'dlr';
```

Comment:
this const could also be typed to defined `sortType`:

```typescript
// sort-value-default.ts
export type sortType = 'r' | 'dmr' | 'dlr' | 'mp';
export const DEFAULT_SORT: sortType = 'dlr';
```

Also - I'd consider changing name of the file to something along the lines of `sort-value.types.ts`